lib/routes/cline/blog.ts

```diff
@@ -7,56 +7,110 @@ const rootUrl = 'https://cline.bot';
 
-            return {
+        // Date extraction from format like "Aug 12, 2024"
+        const dateEl = element.find('div.text-sm.text-gray-500');
+        const dateStr = dateEl.text().trim();
+        const pubDate = dateStr ? parseDate(dateStr) : undefined;
+
+        // Description
+        const summary = element.find('p.text-gray-600').text().trim();
+
+        if (title && link) {
+            articles.push({
                 title,
                 link: fullLink,
                 pubDate,
-                author,
-                description: `<img src="${imgSrc}" /><p>${summary}</p>`,
-            };
-        })
-        .filter((item) => item.title && item.link);
+                author: 'Cline Team',
+                description: `<p>${summary}</p>`,
+            });
+        }
+    });
+
+    // Fallback: try main blog page structure if archive doesn't work
+    if (articles.length === 0) {
+        // Main blog page structure
+        $('article.group').each((_, article) => {
+            const element = $(article);
+
+            const title = element.find('h2, h3').text().trim();
+            const link = element.find('a').first().attr('href');
+            const fullLink = link ? (link.startsWith('http') ? link : `${rootUrl}${link.startsWith('/') ? link : `/${link}`}`) : '';
+
+            // Extract date and author
+            const metaText = element.find('.text-sm.text-slate-500, .text-xs.text-slate-500').text().trim();
+            const dateMatch = metaText.match(/(?:\w+\s*•\s*)?([A-Za-z]+\s+\d{1,2},?\s+\d{4})/);
+            const pubDate = dateMatch ? parseDate(dateMatch[1]) : undefined;
+
+            const authorMatch = metaText.match(/^([^•]+)•/);
+            const author = authorMatch ? authorMatch[1].trim() : 'Cline Team';
+
+            const summary = element.find('p.text-slate-600, p.text-slate-700').text().trim();
+            const imgSrc = element.find('img').attr('src') || '';
+
+            if (title && link) {
+                articles.push({
+                    title,
+                    link: fullLink,
+                    pubDate,
+                    author,
+                    description: imgSrc ? `<img src="${imgSrc}" alt="${title}" /><p>${summary}</p>` : `<p>${summary}</p>`,
+                });
+            }
+        });
+    }
+
+    return articles.filter((item) => item.title && item.link);
 }
 
 async function handler() {
-    // Get blog homepage
-    const response = await got({
-        method: 'get',
-        url: blogUrl,
-    });
+    // Use the archive page which has all articles
+    const archiveUrl = `${rootUrl}/blog/archive`;
 
-    const $ = load(response.data);
+    try {
+        const response = await got({
+            method: 'get',
+            url: archiveUrl,
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (compatible; RSSHub/1.0)',
```

Comment:
Does the site only work when this fixed string of user agent specified instead of RSSHub's default UA which is [randomised Chrome on macOS](https://github.com/DIYgod/RSSHub/blob/d21b8ecf1c88ca44d76685b0e3a885f271ec2b4d/lib/config.ts#L488)